.github/release.yaml

```diff
@@ -2,17 +2,27 @@ changelog:
   exclude:
     labels:
       - ignore-for-release
-    authors:
-      - app/renovate
   categories:
     - title: Breaking Changes 🛠
       labels:
         - Semver-Major
         - breaking-change
```

Comment:
Should we add these labels?

Reply:
I actually decided to cut these categories for now while we are in a pre-release state. Since we have the semantic PRs now, it should be sufficient to just split general changes and dependency updates.